resource/translations/ao_es.ts

```diff
@@ -77,7 +77,8 @@ Razón: %1</translation>
         <location filename="../../src/packet_distribution.cpp" line="651"/>
         <source>You have been banned from the server.
 Reason: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Has sido baneado de este servidor.
```

Comment:
"baneado" sounds so ghetto, is that really how it's said over there

Reply:
Yeah, millennial Spanish is also just English words with "ado", "ando", "endo", etc, at the end. Specially with terms that involve computers. Technically "baneado" is not an actual Spanish word and it's not in the RAE dictionary, so to make it perfect Spanish it would be something like "Has sido prohibido de este servidor" or something like that. But "ban, baneo, banear and baneado" are so extended between anyone who uses computers, and specially between young people that it will actually be clearer if I use "baneado" instead of "prohibido". For reference https://en.wiktionary.org/wiki/banear

Reply:
I grew up with a lot of Spanish slang and we said things like "printear" instead of "imprimir." I thought things were better over there

Reply:
Mexicans tend to mix Spanish and English more, while the Spaniards are typically the most purist with the language. But even the Spaniards borrow many English words. For example they say "freezer" instead of "congelador". Going back to how to translate "You have been banned from the server." I think many websites use the word "bloqueado" (blocked). So it could be translated as "Has sido bloqueado de este servidor." It will be perfect Spanish and should be understandable.

Reply:
It's true that in Spain we usually watch what words we use, and we certainly dont use "printear" instead of "imprimir" (at least i dont). But we do use some english words with the spanish suffixes and call it a day. xD
"Banear" IS one of them. But in this context there are a lot of words that you can use. For example, has sido: "vetado", "echado", "bloqueado",... The one that wind told me you were discussing is "bloqueado". If i had to choose between "bloqueado" and "baneado" i would prefer "baneado" because it's the one that i most commonly say, hear and understand, even if it's not accepted in the dictionary.
Anyway, if you need my assistance for anything else, dont hesitate to ask me! :D